alf/environments/suite_robotics.py

```diff
@@ -166,6 +170,7 @@ def env_ctor(env_id=None):
             from gym.wrappers import FlattenDictWrapper  # pytype:disable=import-error
             env = FlattenDictWrapper(env, keys)
     env = SuccessWrapper(env, max_episode_steps)
+    env = ObservationClipWrapper(env)
```

Comment:
I'm fine with this change, could you at the same time remove the line
suite_robotics.load.gym_env_wrappers=(@ObservationClipWrapper,)
in ddpg_fetchreach.gin?  All the gin files I wrote for robotics env inherits this gin.

Thanks,
Le

Reply:
Removed.